examples/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "paths": {
+      "llamaindex": ["../packages/core/src/index.ts"]
+    }
```

Comment:
Currently, this was my approach to use the local build: https://github.com/run-llama/LlamaIndexTS/blob/844561b6cc2f7418ce3d529c05a2e908ff3a5a72/examples/README.md#using-the-local-build

I tried using `paths` before but was running into issues with multi-file examples (e.g. MongoDB)

If we add this, we should make sure of the following:
1. multi-file examples are running too
2. examples are using the llamaindex package from npm per default (customer use case)
3. update the documentation